opentelemetry-sdk/src/opentelemetry/sdk/_metrics/__init__.py

```diff
@@ -12,31 +12,47 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# pylint: disable=function-redefined,too-many-ancestors
-
-from abc import ABC, abstractmethod
 from atexit import register, unregister
 from logging import getLogger
-from typing import Optional
+from threading import Lock
+from typing import Optional, Sequence
 
 from opentelemetry._metrics import Meter as APIMeter
 from opentelemetry._metrics import MeterProvider as APIMeterProvider
 from opentelemetry._metrics import _DefaultMeter
+from opentelemetry._metrics.instrument import Counter as APICounter
```

Comment:
That approach makes code longer because every time a symbol is to be used, we need to add its module or package as well. It also makes it harder to know if a particular symbol is being used in a module. I tried to use only one `from ... import ...` but `isort` wants it to be separated like this.